kanidmd/src/lib/idm/account.rs

```diff
@@ -66,7 +66,7 @@ macro_rules! try_from_entry {
         let expire = $value.get_ava_single_datetime("account_expire");
 
         let radius_secret = $value
-            .get_ava_single_radiuscred("radius_secret")
+            .get_ava_single_secret("radius_secret")
```

Comment:
Because this is the name of the attribute on the entry that we want to retrieve. It's stored in the db as radius_secret, and has the syntax of "secret string". Look at kanidmd/src/lib/constants/schema.rs line 157 for more.